### PR TITLE
feat: Ethereum RPC call "get_logs"

### PR DESCRIPTION
**Pull request by fkrause98**
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?
There's no functionality to get logs.

Issue Number: [138](https://github.com/keep-starknet-strange/beerus/issues/138)

# What is the new behavior?

- Added functionality to berus_core for the RPC call "get_logs".
- Added CLI command.
- Added rest API endpoint.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

**Comment by danilowhk** [comment]
Could you solve the conflicts?

**Comment by fkrause98** [comment]
> Could you solve the conflicts?

Done!

File: beerus_cli/src/model.rs [hunk 1]
```diff
@@ -106,6 +107,27 @@ pub enum EthereumSubCommands {
...
+        /// Equivalent to from_block = to_block,
+        /// only allowed if neither from_block or to_block
+        /// is supplied.
+        #[arg(short, long, value_name = "BLOCKHASH")]
+        blockhash: Option<String>,
+        /// Starting block to filter from, defaults to "latest".
+        #[arg(short, long, value_name = "FROMBLOCK")]
+        from_block: Option<String>,
+        /// Ending block to filter to, defaults to "latest".
+        #[arg(short, long, value_name = "TOBLOCK")]
```
Comment (danilowhk) [inline beerus_cli/src/model.rs:125]: maybe use "TOP_BLOCK"?

File: beerus_cli/src/model.rs [hunk 2]
```diff
@@ -106,6 +107,27 @@ pub enum EthereumSubCommands {
...
+        /// Address from which the log comes from.
+        #[arg(short, long, value_name = "ADDRESS")]
+        address: Option<String>,
+        /// Equivalent to from_block = to_block,
+        /// only allowed if neither from_block or to_block
+        /// is supplied.
+        #[arg(short, long, value_name = "BLOCKHASH")]
+        blockhash: Option<String>,
+        /// Starting block to filter from, defaults to "latest".
+        #[arg(short, long, value_name = "FROMBLOCK")]
```
Comment (danilowhk) [inline beerus_cli/src/model.rs:122]: maybe use "FROM_BLOCK"?

**Review by danilowhk — APPROVED** [review]
Lgtm!

**Pull request merged**